pyproject.toml

```diff
@@ -18,14 +18,11 @@ requires = [
     "setuptools>=64.0.0",
     "cython>=0.29,<0.30",
     "tomli; python_version < '3.11'",
-    "versioneer>=0.24",
 ]
 
 [project]
 name = "ucx-py"
-dynamic = [
-    "version",
-]
+version = "0.31.0"
```

Comment:
Sorry, I was OOO and am only seeing this now. I'm wondering here if it's possible to programmatically get the version, as it was done before, maybe by looking at the branch name? If not, who's responsible for keeping this up-to-date, is it automatically managed by CI or do we have to do it manually?

Reply:
Think the idea is that [the `update-version.sh` script]( https://github.com/rapidsai/ucx-py/blob/branch-0.31/ci/release/update-version.sh ) would handle this

Reply:
Ah ok, got it now. Thanks John for clarifying!

Reply:
Yup that's right, thanks for answering here John. Long term I am hoping to do this even more programatically than we do now, but that has to do with improvements to update-version.sh and how the version there is produced.